source/MapPanel.h

```diff
@@ -144,7 +144,13 @@ class MapPanel : public Panel {
 
 	// Distance from the screen center to the nearest owned system,
 	// for use in determining which governments are in the legend.
-	std::map<const Government *, double> closeGovernments;
+	std::map<const Government *, unsigned int> bigGovernments;
```

Comment:
```suggestion
	// The number of systems owned by each government which are currently
	// on the screen, for use in determining which governments are in the legend.
	std::map<const Government *, unsigned int> governmentCounts;
```

Reply:
Implemented, but using a commit so as not to mess up my branch